test/sagemaker_tests/tensorflow/inference/test/integration/local/test_multi_model_endpoint.py

```diff
@@ -188,6 +188,49 @@ def test_load_two_models():
     assert len(res3) == 2
 
 
+@pytest.mark.model("half_plus_three, half_plus_two")
```

Comment:
Would there be any benefit to using a bigger model?
Just thinking about the stress test that was done last week

Reply:
i think there'd be benefit. subsequent pr?